docker/build/installers/install_drivers_deps.sh

```diff
@@ -28,7 +28,7 @@ bash /tmp/installers/install_proj4.sh
 
 # Required by audio-driver python version
 apt-get -y update && \
-    apt-get -y install python3-pyaudio
+    apt-get -y install python3-pyaudio portaudio19-dev
```

Comment:
@Crysple a new docker might be needed for this pr, right? @xiaoxq @lfcarol @storypku

Reply:
> @Crysple a new docker might be needed for this pr, right? @xiaoxq @lfcarol @storypku

Yep.

Reply:
> > @Crysple a new docker might be needed for this pr, right? @xiaoxq @lfcarol @storypku
> 
> Yep.

Could you help with the new docker please ?

Reply:
> Could you help with the new docker please ?

No problem.